.gitattributes

```diff
@@ -6,6 +6,7 @@
 # Mark lock files as generated to avoid diffing
 pnpm-lock.yaml linguist-generated
 package-lock.json linguist-generated
```

Comment:
```suggestion
```

Reply:
They are used in test fixures, that's why they're there.